db/migrate/20170430050223_backfill_library_timestamps.rb

```diff
@@ -0,0 +1,11 @@
+class BackfillLibraryTimestamps < ActiveRecord::Migration
+
+  def change
+    LibraryEntry.where("progress > 0").find_in_batches do |batch|
```

Comment:
yeah I tried it like that first and it didn't work, which is why I switched to `find_in_batches`